content/kubeflow/pipelines.md

```diff
@@ -14,11 +14,25 @@ Kubeflow Pipeline is one the core components of the toolkit and gets deployed au
 * An SDK for defining and manipulating pipelines and components.
 * Notebooks for interacting with the system using the SDK.
 
+#### Kubeflow Python SDK
```

Comment:
I feel this will cause confusion for users because when I think about it, Python SDK and Sagemaker pipelines are two independent chapters. I think there is a better way to highlight other notebooks. Let's think about clear instructions on how to highlight other notebooks but provide a walkthrough for single notebook

Reply:
I don't quite understand. They are both under KFP chapter. Which notebook you mean you want to highlight?

Reply:
We will have similar issue later if we pick more notebooks for one component. Let me know how can I reorganize the structure.